### DISCUSSION TITLE
[Question][Hw7] About schedule of last homework (HW7)

### DISCUSSION THREAD

**Discussion by JOYB28** [prosyslab-classroom/cs348-information-security#726] [Q&A]
Name: Yoonseo Kim

I wonder when the last assignment will be released and when the due will be.
I wonder if due overlaps with final exam week.

과제가 기말고사 기간과 겹치는지 궁금합니다.

**Comment by yeonhee-ryou** [comment]
It will be announced tomorrow and the due will be June 16th considering the exam period

**Comment by JOYB28** [comment]
Thanks for the reply! 🙏
